RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -7,12 +7,168 @@
 
 import SwiftUI
 
+/// Player control
 struct ContentView: View {
+    // MARK: - Private Constants
+    private enum Constants {
+        static let emptyString = ""
+        static let posterWidthHeight: CGFloat = 250
+        static let posterCornerRadius: CGFloat = 50
```

Comment:
```suggestion
        static let posterWidthHeight = 250
        static let posterCornerRadius = 50
```
Как-то мне ставили коммент на то, чтобы я в константах не указывал явно тип, тоже момент очень спорный если речь идет о ширине и высоте, т.к она используется в CGFloat, но на всякий случай...